docs/cookbook/migrations/homebrew.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Homebrew
+description: Common questions and solutions for using Rust with Flox
+---
+
+# Homebrew migration guide
+Flox can replace Homebrew entirely, or they can be used together.
+
+This guide explains how to introduce Flox into environments where Homebrew is currently being used, either as a replacement or an addition. It introduces new concepts and proposes a basic procedure for mapping packages.
+
+## Why you might want to migrate
+Homebrew does a great job, and has been loved as the “missing package manager” for a generation of macOS users. But there are a few reasons you might consider moving to Flox:
+
+* **You want to install packages into discrete environments.** Flox allows you to create as many environments as you need, each with a different set of packages.
+* **You need a reproducible manifest.** Flox environments are defined in a TOML manifest that can be shared and reproduced - often across OS and system architectures.
+* **You need a broader set of packages.** Homebrew has a sizeable collection of packages, but the Flox Catalog is based on Nixpkgs, the largest collection of packages in the world.
+* **You need older versions of packages.** The Flox Catalog contains historical versions of each of its packages - in many cases, going back decades.
+
+## New concepts
+Flox works similarly to Homebrew, but there are a few new concepts to be aware of.
```

Comment:
I would say "Flox feels similar to Homebrew" since it doesn't actually work anything like Homebrew under the hood. I think the similarities are mainly "you have install, uninstall, and search commands".